crates/bevy_render/src/render_resource/storage_buffer.rs

```diff
@@ -101,6 +104,16 @@ impl<T: ShaderType + WriteInto> StorageBuffer<T> {
         self.label.as_deref()
     }
 
+    /// Set the buffer usage of the buffer.
```

Comment:
```suggestion
    /// Adds more [`BufferUsages`] to the buffer.
```

The same change should be done for the other instances of this function.